data/schema.json

```diff
@@ -0,0 +1,460 @@
+          
+        },
+        {
+          "type": "array", 
+          "minItems": 2,
+          "items": {
+            "$ref": "#/definitions/licenseType"
+          }
+        }
+      ]
+    },
+    "summary": {
+      "description": "Brief package summary.",
+      "anyOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "array", 
+          "minItems": 2,
+          "items": {
+            "type": ["string", "object"]
+          }
+        }
+      ]
+    },
+    "component": {
+      "description": "Component / group of packages this package belongs to. Check available components via `eopkg lc`.",
+      "anyOf": [
+        {
+          "$ref": "#/definitions/componentType"
+        }, 
+        {
+          "type": "array", 
+          "minItems": 2,
+          "items": {
+            "anyOf": [
+              {
+                "$ref": "#/definitions/componentType"
+              },
+              {
+                "type": "object", 
+                "additionalProperties": {
+                  "$ref": "#/definitions/componentType"
+                }
+              }
+            ]
+          }
+        }
+      ]
+      
+    },
+    "description": {
+      "description": "More extensive description of the software, usually taken from the vendor website.",
+      "anyOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "array", 
+          "minItems": 2,
+          "items": {
+            "type": ["string", "object"]
+          }
+        }
+      ]
+    },
+
+    "autodep": {
+      "type": "boolean",
+      "description": "Set to 'false' to disable automatic binary dependency resolution at build time."
+    },
+    "avx2": {
+      "type": "boolean",
+      "description": "Set to 'true' to build package with the x86-64-v3 microarchitecture."
+    },
+    "build": {
+      "type":"string",
+      "description": "Use this step to run the build portion, for example, 'make'.",
+      "examples": ["%make"]
+    },
+    "builddeps": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "Build dependencies for the package.",
+      "uniqueItems": true
+    },
+    "ccache":{
+      "type": "boolean",
+      "description": "Set to 'false' to disable ccache."
+    },
+    "check": {
+      "type":"string",
+      "description": "This is where tests / checking should occur, for example, 'make check'."
+    },
+    "checkdeps": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "Build dependencies which will *not* be considered when determining build order for automatic builds. These dependencies should only be used during the 'check' build phase.",
+      "uniqueItems": true
+    },
+    "clang": {
+      "type": "boolean",
+      "description": "Set to 'true' if this package benefits from being built with Clang."
+    },
+    "conflicts": {
+      "type": "string",
```

Comment:
This can be either a single string, or an array of strings. Actually, grepping for it in the packages repository, we can also set patterned packages to have conflicts. That's gonna be complicated...